docs/source/dataanalysis/dataanalysis.rst

```diff
@@ -93,6 +93,8 @@ The code writes out the values in an ASCII file prefixed ``reduced_beam_characte
     Horizontal and vertical dispersion (unit: meter)
 * ``dispersion_px``, ``dispersion_py``
     Derivative of horizontal and vertical dispersion (unit: dimensionless)
+* ``emittance_1``, ``emittance_2``, ``emittance_3``
```

Comment:
We need to update `usage/parameters.rst` and `usage/python.rst` to advertise the inputs control to calculate these.

Here, we might need to add one more line referencing these sections, so users know these entries are only added conditionally.